File: JavaScript Core/JS Applications/10.Exam-MessageApp/src/Views/RegisterView.js
import React, { Component } from 'react';

export default class RegisterView extends Component {
    render() {
        return (
            <div id="viewRegister">
                <h1>Please register here</h1>
                <form id="formRegister" onSubmit={this.submitForm.bind(this)}>
                    <label>
                        <div>Username:</div>
                        <input type="text" name="username" id="registerUsername" required ref={e => this.usernameField = e} />
                    </label>
                    <label>
                        <div>Password:</div>
                        <input type="password" name="password" id="registerPasswd" required ref={e => this.passwordField = e} />
                    </label>
                    <label>
                        <div>Name:</div>
                        <input type="text" name="name" id="registerName"/>
                    </label>
                    <div>
                        <input type="submit" value="Register"/>
                    </div>
                </form>
            </div>
        );
    }

    submitForm(event) {
        event.preventDefault();
        this.props.onsubmit(
            this.usernameField.value, this.passwordField.value);
    }
}

File: C# Web/C# Web Development Basics/07.Web Server - Asynchronous Processing - Exercise - Concurrent Users/WebServer/Application/Views/User/RegisterView.cs
namespace WebServer.Application.Views.Register
{
    using WebServer.Server.Contracts;

    public class RegisterView : IView
    {
        public string View()
        {
            return "<body>" +
                   "     <form method=\"POST\">" +
                   "         Name</br>" +
                   "         <input type=\"text\" name=\"name\" /><br/>" +
                   "         <input type=\"submit\" />" +
                   "     </form>" +
                   "</body>";
        }
    }
}